.ci/azure/linux.yml

```diff
@@ -99,3 +111,24 @@ jobs:
     env:
       LD_LIBRARY_PATH: $(BIN_DIR)/lib
     displayName: 'Java tests'
+
+  - script: |
+      sudo apt-get install -y python3-setuptools
+      python3 -m pip install \
+          defusedxml>=0.5.0 \
+          networkx>=1.11 \
+          test-generator==0.1.1 \
+          torch==1.7.0 \
+          torchvision==0.8.1 \
+          opencv-python==4.4.0.46
```

Comment:
In order to avoid commits to pipeline when dependencies will change, I can recommend to create requirements.txt file in test folder and reference it here

Reply:
Done

Reply:
Done